### ISSUE TITLE
Link hover color in footer fails color contrast

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#114]
color contrast is currently 4.494:1. Change hover color to#006365.
